v22.1/take-full-and-incremental-backups.md

```diff
@@ -92,45 +153,72 @@ Periodically run the [`BACKUP`][backup] command to take a full backup of your cl
+`RESTORE` will re-validate [indexes](indexes.html) when [incremental backups](take-full-and-incremental-backups.html) are created from an older version (v20.2.2 and earlier or v20.1.4 and earlier), but restored by a newer version (v21.1.0+). These earlier releases may have included incomplete data for indexes that were in the process of being created.
 {{site.data.alerts.end}}
 
 ## Incremental backups with explicitly specified destinations
 
-To explicitly control where your incremental backups go, use the [`INTO {subdirectory} IN (destination)`](backup.html#synopsis) syntax:
+<span class="version-tag">New in v22.1:</span> To explicitly control where your incremental backups go, use the [`incremental_location`](backup.html#options) option. By default, incremental backups are stored in the `/incrementals` subdirectory at the root of the collection. However, there are some advanced cases where you may want to store incremental backups in a different storage location.
+
+In the following examples, the `{collectionURI}` specifies the storage location containing the full backup. The `{explicit_incrementalsURI}` is the alternative location that you can store an incremental backup:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> BACKUP DATABASE bank INTO '{subdirectory}' IN '{destination}' \
-    AS OF SYSTEM TIME '-10s' \
-    WITH revision_history;
+BACKUP INTO LATEST IN '{collectionURI}' AS OF SYSTEM TIME '-10s' WITH incremental_location = '{explicit_incrementalsURI}';
 ~~~
 
-A full backup must be present in the `destination` for an incremental backup to be stored in a `subdirectory`. If there isn't a full backup present in the `destination` when taking an incremental backup, one will be taken and stored in the `destination`.
+Although the incremental backup will be in a different storage location, it is still part of the logical [backup collection](#backup-collections).
 
-{{site.data.alerts.callout_info}}
-To take incremental backups, you need an [Enterprise license](enterprise-licensing.html).
-{{site.data.alerts.end}}
+A full backup must be present in the `{collectionURI}` in order to take an incremental backup to the alternative `{explicit_incrementalsURI}`. If there isn't a full backup present in `{collectionURI}` when taking an incremental backup with `incremental_location`, the error `path does not contain a completed latest backup` will be returned.
+
+For details on the backup directory structure when taking incremental backups with `incremental_location`, see this [incremental location directory structure](#incremental-location-structure) example.
+
+<a name="backup-earlier-behavior"></a> To take incremental backups to a subdirectory in the `collectionURI` that you define (rather than the default `/incrementals` directory), use the `incremental_location` option:
+
+~~~ sql
+BACKUP INTO LATEST IN '{collectionURI}' AS OF SYSTEM TIME '-10s' WITH incremental_location = '{collectionURI/subdirectory}';
```

Comment:
i wonder if this example should get moved to the "callout_info" blurb for pre 22.1 behavior? If you don't think the example should moved, perhaps mention that this example is for folks that want to mimic pre 22.1 behavior.

also the the incremental_location should be '{collectionURI}', instead of '{collectionURI/subdirectory}'. The LATEST keyword will handle the subdirectory resolution.

Reply:
So if you want to mimic old behavior: `BACKUP INTO LATEST IN {collectionURI}` or `BACKUP INTO {subdirectory} in {collectionURI}`... isn't `BACKUP INTO LATEST IN '{collectionURI}' AS OF SYSTEM TIME '-10s' WITH incremental_location = '{collectionURI}'` .... if you don't specify a subdir on the second `collectionURI` (which is the same as the first because we're trying to mimic the old behavior), will `incremental_location` create a subdir that isn't `/incrementals` in v22.1?

Reply:
OK so I tested again, and if you just specify the same `collectionURI` with no `/subdir` the incremental will just go into a date-based path under the full backup.... it won't go into the `/incrementals`. 

If you then run `BACKUP DATABASE movr INTO LATEST IN {collectionURI}` it will continue with the old behavior, it will won't then create a default `/incrementals` directory... I assume to maintain the chain in some way. Expected behavior? If so, I will add a sentence to this effect now.

Reply:
yeah this is expected and real subtle:

If the backup already contains incrementals using the pre 21.2 style (i.e. inceremtals in the full backup dir), in 22.1 `BACKUP INTO LATEST IN '{collectionURI}' ` will store the incrementals by default in the  full dir. this is to maintain backward compatibility for users trying to maintain a backup chain across versions.

Now, if the user in 22.1 runs `BACKUP INTO LATEST IN '{collectionURI}' ` and the full backup does _not_ contain incrementals in that full dir, we default to storing the incrementals in `./incrementals` dir. 

So if the user is trying to mimic the old behavior for, the full proof method is to use the `incremental_location` parameter.